### ISSUE TITLE
GO:0070994 detection of oxidative stress (MF?)

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#12891]
The series of events in which a stimulus indicating oxidative stress is received and converted into a molecular signal.

is this describing an "oxidative stress sensor" ? (molecular function).
The term name sounds like a sensor, but I'm confused by the "series of events" part which sounds like it is describing an entire signal transducation pathway. 

This has only 2 EXP annotations  (one pombe) and this seems to be an  "sensor"

There is  a precedent for "sensor activity" terms

 GO:0032935 sterol sensor activity
Interacting selectively and non-covalently with and responding, e.g. by conformational change, to changes in the cellular level of a sterol.

Should the sensor be defined functionally as initiating some response?

**Comment by ukemi** [comment]
I think the detection would be the pathway(s) upstream of another downstream pathway such as autophagy or apoptosis. See PMID:23849868 as an example.

Detection and the execution of the downstream pathway would lead to a process such as GO:0097468 (programmed cell death in response to reactive oxygen species).

**Comment by ukemi** [comment]
We discussed something very similar to this today on the annotation call regarding a model about sensing oxygen. I think the major issue was how to describe the function of a sensor. In the case of this model, an enzyme that used the oxygen as a substrate was also playing the role of the oxygen sensor. There was discussion about how to distinguish this from all other enzymatic functions because they in effect serve a 'sensors' for their substrates. 

The above comments still hold with respect to the distinction between a putative function and the process.

**Comment by vanaukenk** [comment]
So wrt EGL-9 activity, the stimulus is actually normal levels of O2.  In that case, for the detection term, would the molecular signal be the hydroxylated HIF-1 transcription factor that is the output of EGL-9's enzymatic activity?

**Comment by ukemi** [comment]
So this one is not a stress, but detection of normal levels I guess. Maybe this is a good example of why it would be a process. Perhaps both the binding of the oxygen through the hydroxylated TF is the detection phase. It will be nice to discuss this one again on an annotation call.

**Comment by ValWood** [comment]
I was looking at some of the pombe annotations in this area during the call and comparing to Kimberley's model.  This is a process  I have flagged  for LEGO because PomBase used the most unwieldy terms, and the pathway is well characterised in pombe.

Check out the oxygen ofd1 sensor
http://www.pombase.org/spombe/result/SPBC6B1.08c#LitRef-22017871
(I think this is EGL-9 ortholog?). If yours and @mah11   LEGO exercise is a success you might wan to try to co- model for this process? 

Basically if I remember correctly ofd1 senses both high and low oxygen (Hypoxia and hyperoxia), and has  multiple mechanisms of regulating the  signalling pathway (either positive regulation of TF catabolism, or negative/positive regulation of DNA binding depending if oxygen levels are high or low. I think that although ofd1 is the sensor in both cases sometimes the sensor requires hydroxylation and sometimes not.

A single group (Peter Espenshade's)  published all of the pombe papers, and Peter and his lab curated most of these papers (although the unwieldy terms are largely mine- I think some could well be streamlined or removed). Peter is very helpful and responsive and would be a good person to ask about the essence of the "sensor" activity.

His lab have even done some mathematical modelling of this process where they lay out the network deduce that the catabolic part of this pathway  likely to be regulating the  turning off of the hypoxic response when oxygen is restored:
https://www.ncbi.nlm.nih.gov/pubmed/22833559

It would be interesting to see  in this case, how the LEGO model translates into gene page annotations (an exercise I still intend to follow through for my cohesin model when I get chance). 

v

**Comment by mah11** [comment]
> If yours and @mah11 LEGO exercise is a success you might wan to try to co- model for this process?

??? The model Kimberly and I are about to work on is on totally unrelated biology.

**Comment by ValWood** [comment]
I know...

**Comment by mah11** [comment]
then why bring it up on this ticket?

**Comment by ukemi** [comment]
So is everything clear on this ticket then? Can we close it? The bottom line is that in some systems, the detection of oxidative stress can be thought of as a process/pathway that triggers further downstream action that is part of the response.

**Comment by ValWood** [comment]
> then why bring it up on this ticket?

Err because that is what the ticket is about? 

> So is everything clear on this ticket then? Can we close it? 

It isn't clear to me yet, but I'm sure it will become clearer during modelling.... so yes...

**Comment by mah11** [comment]
>> then why bring it up on this ticket?
> 
> Err because that is what the ticket is about? 

My entire point is that Kimberly and I plan to work on a model that this ticket is most assuredly _not_ about.

**Comment by ValWood** [comment]
But this one would be useful?

**Comment by mah11** [comment]
So? That doesn't make bringing up an unrelated model relevant.

**Comment by vanaukenk** [comment]
Hi,
There are some aspects of the distinction between detection/sensor terms that are still not quite clear to me, but I think this really needs some more in-depth work (read: mini-project) and looking at various ways that sensor is used by biologists to see if we can perhaps define this term better or make sure that if we decide not to use it, we aren't missing important aspects of the biology.
I was thinking about the unfolded protein response as an analogous process, where some resident ER proteins are described as sensors, but we likely have just annotated them to 'unfolded protein binding' as we don't have a term 'unfolded protein sensor'.
So, bottom line, I think we could give some more thought to how we want to represent this biology and that thought will be aided by LEGO modeling of these processes in different organisms.
David, do you want to flag this with a mini-project label?

**Comment by ukemi** [comment]
I have tagged this with mini-project. One question that keeps popping into my mind based on @hdrabkin comments on the call yesterday. How to we distinguish a sensor in a meaningful way? Are all receptors classified as sensors? Is any rate limiting enzymatic activity in a pathway a sensor? I'm not against having a term because I think the community thinks about 'sensors', but I don't want to open a can of worm down the road because we haven't thought it out well enough.

**Comment by ValWood** [comment]
I have a nice example for "oxygen sensor" in signalling.
glycolytic enzyme acts as a sensor of oxygen , Tdh1 GAPDH is transiently oxidized, which enhances the association of Tdh1 with Mcs4 (upstream of MAPK signalling)
http://pombase2.aska.gen.nz/reference/PMID%3A18406331

Currently we annotated to 
http://pombase2.aska.gen.nz/term/GO%3A0070994
but I'd like to  capture this as
MF "oxygen sensor" part of "positive regulation of phosphorelay signal transduction system"

(or even part of the phosphorelay system, depending whether or not the sensor is included)

**Comment by pgaudet** [comment]
Looking at this example with @thomaspd 
How about 'hydrogen-peroxide-activated adaptor activity' 
(and H2O2 sensor as a broad synonym- because there may be other response mechanisms)

ie its function is to bridge the interaction between two proteins. Unlike examples in #16747 
where the small molecule activates a guanylate cyclase or an adenylate cyclase. 

Pascale

**Comment by ValWood** [comment]
I don't think sensors are adaptors though? They aren't bridging between 2 proteins, only enhancing the binding of one protein to another (as far as I'm aware). Usually, they just result in a conformational change (act as a switch I guess).

An adaptor is physically linking 2 proteins, and for a sensor the critical part  of the activity  is the "detection".

What's the objection to using "sensor" in the term name?

**Comment by pgaudet** [comment]
Hi @ValWood 

What needs to be done here ? 
You want 'H2O2 sensor activity'? 
(see https://github.com/geneontology/go-ontology/issues/16747)

**Comment by ValWood** [comment]
I don't think any further action is required and that this can close.
Now we have a sensor grouping term, we can request new terms as required...

**Comment by pgaudet** [comment]
ok great !
